static/.htaccess

```diff
@@ -8,503 +8,21 @@ Options -Indexes -MultiViews
-RewriteRule ^docs/next/components/modeler/web-modeler/advanced-modeling/manage-connector-templates/(.*)$ /docs/next/components/connectors/manage-connector-templates/$1 [R=301,L]
+RewriteRule ^docs/self-managed/overview/(.*)$ /1.3/docs/self-managed/about-self-managed/$1 [R=301,L]
+RewriteRule ^docs/self-managed/overview$ /1.3/docs/self-managed/about-self-managed/ [R=301,L]
 
-# connectors restructure 8.0-8.2
-RewriteRule ^docs/8.0/components/connectors/out-of-the-box-connectors/rest/(.*)$ /docs/8.0/components/connectors/protocol/rest/$1 [R=301,L]
-RewriteRule ^docs/8.0/components/connectors/out-of-the-box-connectors/graphql/(.*)$ /docs/8.0/components/connectors/protocol/graphql/$1 [R=301,L]
-RewriteRule ^docs/8.0/components/connectors/out-of-the-box-connectors/http-webhook/(.*)$ /docs/8.0/components/connectors/protocol/http-webhook/$1 [R=301,L]
-RewriteRule ^docs/8.0/components/modeler/web-modeler/advanced-modeling/manage-connector-templates/(.*)$ /docs/8.0/components/connectors/manage-connector-templates/$1 [R=301,L]
-RewriteRule ^docs/8.1/components/connectors/out-of-the-box-connectors/rest/(.*)$ /docs/8.1/components/connectors/protocol/rest/$1 [R=301,L]
-RewriteRule ^docs/8.1/components/connectors/out-of-the-box-connectors/graphql/(.*)$ /docs/8.1/components/connectors/protocol/graphql/$1 [R=301,L]
-RewriteRule ^docs/8.1/components/connectors/out-of-the-box-connectors/http-webhook/(.*)$ /docs/8.1/components/connectors/protocol/http-webhook/$1 [R=301,L]
-RewriteRule ^docs/8.1/components/modeler/web-modeler/advanced-modeling/manage-connector-templates/(.*)$ /docs/8.1/components/connectors/manage-connector-templates/$1 [R=301,L]
-RewriteRule ^docs/8.2/components/connectors/out-of-the-box-connectors/rest/(.*)$ /docs/8.2/components/connectors/protocol/rest/$1 [R=301,L]
-RewriteRule ^docs/8.2/components/connectors/out-of-the-box-connectors/graphql/(.*)$ /docs/8.2/components/connectors/protocol/graphql/$1 [R=301,L]
-RewriteRule ^docs/8.2/components/connectors/out-of-the-box-connectors/http-webhook/(.*)$ /docs/8.2/components/connectors/protocol/http-webhook/$1 [R=301,L]
-RewriteRule ^docs/8.2/components/modeler/web-modeler/advanced-modeling/manage-connector-templates/(.*)$ /docs/8.2/components/connectors/manage-connector-templates/$1 [R=301,L]
-RewriteRule ^docs/components/connectors/out-of-the-box-connectors/rest/(.*)$ /docs/components/connectors/protocol/rest/$1 [R=301,L]
-RewriteRule ^docs/components/connectors/out-of-the-box-connectors/graphql/(.*)$ /docs/components/connectors/protocol/graphql/$1 [R=301,L]
-RewriteRule ^docs/components/connectors/out-of-the-box-connectors/http-webhook/(.*)$ /docs/components/connectors/protocol/http-webhook/$1 [R=301,L]
-RewriteRule ^docs/components/modeler/web-modeler/advanced-modeling/manage-connector-templates/(.*)$ /docs/components/connectors/manage-connector-templates/$1 [R=301,L]
+RewriteRule ^docs/components/overview/(.*)$ /1.3/docs/components/components-overview/$1 [R=301,L]
```

Comment:
It will be important to test these redirects out thoroughly for archived versions, because as this example proves, they might no longer be correct! (This one redirects to a 404.)

Reply:
I would assume most redirects for unmaintained versions can be deleted. Or am I missing something?

Reply:
I suppose we could just delete them because they're _unsupported_, but it doesn't seem like much effort to me to preserve them, if it means helping some people who were holding on to links before we archived. Is this blocking?

Reply:
Nope!